tests/unit/introspection.tcl

```diff
@@ -61,6 +65,26 @@ start_server {tags {"introspection"}} {
         $rd4 close
     }
 
+tags {"needs:debug"} {
+    test {CLIENT KILL AGE will kill old clients} {
+            set rd1 [redis_deferring_client]
+            r debug sleep 2
+            set rd2 [redis_deferring_client]
+            set connected_clients [s connected_clients]
+            assert {$connected_clients >= 3}
+            # Should kill everything except r and rd2
+            set res [r client kill age 1]
+            assert {$res == $connected_clients - 2}
+
+            # rd2 should still be connected
+            $rd2 ping
+            assert_equal "PONG" [$rd2 read]
+
+            $rd1 close
+            $rd2 close
+        }
+} ;# tag
```

Comment:
you can use this pattern:
```
test {name} {
    code
} {} {tag}
```
i.e. the last two arguments are expected return value and tag.